src/main/scala/temple/generate/server/go/service/main/GoServiceMainHandlersGenerator.scala

```diff
@@ -19,12 +19,16 @@ import scala.Option.when
 
 object GoServiceMainHandlersGenerator {
 
-  private def generateResponseMapFormat(attributeType: AttributeType): String =
-    // Must add formatting to attributes with datetime type
-    attributeType match {
-      case DateTimeType => s".${genFunctionCall("Format", "time.RFC3339")}"
+  private def generateResponseMapFormat(attributeType: AttributeType): String = {
+    // Must add formatting to attributes with date, time or datetime type
+    val layout = attributeType match {
+      case DateType     => doubleQuote("2020-12-31")
```

Comment:
@smithwjv See my gist (https://gist.github.com/jaylees14/e86b262ca4e0b8185cb5357c78fc01f3) for the correct format here. You need to use the reference time: https://gobyexample.com/time-formatting-parsing

Reply:
Was wondering how they told the difference between days and months aha